src/Patch/Element.js

```diff
@@ -116,6 +128,53 @@ export default function(internals) {
     });
   }
 
+  function patch_outerHTML(destination, baseDescriptor) {
+    patch_HTMLsetter(destination, 'outerHTML', baseDescriptor, (node, patchFunction, {parentNode, previousSibling, nextSibling}) => {
+      if (previousSibling === null && nextSibling === null) {
+        patchFunction.call(internals, parentNode);
+      } else {
+        let sibling = previousSibling && previousSibling.nextSibling || parentNode.firstChild;
+
+        while (sibling !== null && sibling !== nextSibling) {
+          patchFunction.call(internals, sibling);
+          sibling = sibling.nextSibling;
+        }
+      }
```

Comment:
I think this needs another case at the beginning that checks if `parentNode === null`: in that case, there's no need to call `patchFunction` with `parentNode` (which would otherwise happen because `previousSibling === null && nextSibling === null`).